README.md

```diff
@@ -16,6 +19,10 @@ More specs to come soon!
 
 These test-plans historically used Testground. To read why we're now using `docker compose` instead please see: [Why we're moving away from Testground](https://github.com/libp2p/test-plans/issues/103)
 
+## Performance Benchmarking
+
+Please see the [libp2p perf protocol](https://github.com/libp2p/specs/blob/master/perf/perf.md) and the [benchmarking README](https://github.com/libp2p/test-plans/tree/master/perf#libp2p-performance-benchmarking).
```

Comment:
Note that the `perf/README.md` already points to the libp2p perf protocol specification. Slight preference for not linking to the specs repository twice.